Dockerfile

```diff
@@ -1,16 +1,15 @@
-# Docker image for the Drone SFTP plugin
-
 FROM alpine:3.3
 
 RUN apk update && \
-    apk add nodejs && \
-    apk add git && \
```

Comment:
Yeah but the image used for the build does have git installed: 

![selection_066](https://cloud.githubusercontent.com/assets/661901/12714140/c4703c82-c8ca-11e5-801b-a54bcf46d1f0.png)